tools/ci_build/github/apple/coreml_supported_ops.md

```diff
@@ -1,34 +1,38 @@
 <!--
 Keep in sync with doco generated from /docs/execution-providers/CoreML-ExecutionProvider.md on the gh_pages branch
 -->
-|Operator|Note|
-|--------|------|
-|ai.onnx:Add||
-|ai.onnx:ArgMax||
-|ai.onnx:AveragePool|Only 2D Pool is supported.|
-|ai.onnx:BatchNormalization||
-|ai.onnx:Cast||
-|ai.onnx:Clip||
-|ai.onnx:Concat||
-|ai.onnx:Conv|Only 1D/2D Conv is supported.<br/>Weights and bias should be constant.|
-|ai.onnx:DepthToSpace|Only DCR mode DepthToSpace is supported.|
-|ai.onnx:Div||
-|ai.onnx:Flatten||
-|ai.onnx:Gemm|Input B should be constant.|
-|ai.onnx:GlobalAveragePool|Only 2D Pool is supported.|
-|ai.onnx:GlobalMaxPool|Only 2D Pool is supported.|
-|ai.onnx:LRN||
-|ai.onnx:MatMul|Input B should be constant.|
-|ai.onnx:MaxPool|Only 2D Pool is supported.|
-|ai.onnx:Mul||
-|ai.onnx:Pad|Only constant mode and last two dim padding is supported.<br/>Input pads and constant_value should be constant.<br/>If provided, axes should be constant.|
-|ai.onnx:Pow|Only supports cases when both inputs are fp32.|
-|ai.onnx:PRelu|Input slope should be constant.<br/>Input slope should either have shape [C, 1, 1] or have 1 element.|
-|ai.onnx:Relu||
-|ai.onnx:Reshape||
-|ai.onnx:Resize||
-|ai.onnx:Sigmoid||
-|ai.onnx:Squeeze||
-|ai.onnx:Sub||
-|ai.onnx:Tanh||
-|ai.onnx:Transpose||
+| Operator                   | Note                                                                                                                                                      |
+|----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ai.onnx:Add                ||
```

Comment:
This formatting change would break [this](https://github.com/microsoft/onnxruntime/blob/9db24f8fec40c047044094441c90bedfe05bae6f/tools/python/util/mobile_helpers/usability_checker.py#L35-L48) tool

Reply:
Thanks for the heads up; IntelliJ was trying to be a smart alec 🤦. I've reverted the formatting & the diff is much cleaner now.